index.html

```diff
@@ -12,12 +12,12 @@
     <title>Portfolio</title>
 </head>
 <body>
-    
+
```

Comment:
# Requirements missing are in the red box :blush:

![missing (copy)](https://user-images.githubusercontent.com/93039359/178511954-83e93cd6-3c58-40ec-80ef-dc8f8827a3a8.png)


- [ ] It's good that you have added the summary and added some points of accessibility changes but you didn't follow the requirements completely. You need to add all the points which are also required in the requirements but you have added some points. You can take a look at this [pull request summary](https://github.com/orozCoding/myportfolio/pull/8) to get a good idea :+1: